package.json

```diff
@@ -58,7 +50,8 @@
     "test-coverage": "node ./out/test/runTest.js --coverage",
     "prettier:write": "prettier --write \"src/**/*.{ts, js}\"",
     "prettier:verify": "prettier --list-different \"src/**/*.{ts, js}\"",
-    "vscode:prepublish": "npm run pretest && npm run test"
+    "bundle:extension": "esbuild ./src/extension.ts  --bundle --outdir=out --format=cjs --target=es2020 --platform=node --external:vscode --external:@salesforce/core --external:@salesforce/lwc-dev-mobile-core --external:@salesforce/sf-plugins-core --minify",
+    "vscode:prepublish": "npm run clean && npm run bundle:extension && npm prune --omit=dev"
```

Comment:
Shouldn't `npm prune --omit=dev` happen _before_ the `bundle:extension` step? Or otherwise if that doesn't work with stuff put together all in this one prepublish step, perhaps this step should be broken out to first build, then omit, and _then_ bundle.

Otherwise, I don't see the prune step doing anything for us here.

Reply:
I thought the same thing yesterday, but i believe what is happening is prepublish runs, then bundling occurs, THEN the packaging occurs. If I change these around, the prune will remove all dev-deps and then `esbuild` wont even be found. I also tested it yesterday by adding a `&& ls -la` to the end of `bundle:extension` and no *.vsix file exists after esbuild executes.

Reply:
Yeah, I figured simply changing the order would probably break things. I think you'd need to do a build step, then an omit step, and then a bundle step. Right now the "build and bundle" is all merged together in one command, so it's not possible.

It's not a big deal though, either way. I'm okay leaving it as is for this release.